README.md

```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
 
 DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing.
 
-### Build a local docker container
+If using GCP, there are [pre-built images](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) which come with CUDA, docker, and nvidia-docker pre-installed: 
```

Comment:
I don't think it should end with a colon. maybe a comma?

I'm also not sure if the spacing looks right, maybe it should be one big paragraph?  (Maybe not, they are different ideas).  Can you convince me one way or the other?

Reply:
I think it's basically one idea, single paragraph is probably better